### PR TITLE
21 bug not sending cmcd data on manifest request

### PR DESCRIPTION
**Pull request by coatesjuan**

### CODE REVIEW

**Review by mcamardon — APPROVED** [review]

File: README.md
Comment (nicolaslevy) [inline README.md:1]: We should add something explaining how to retrieve sessionid and contentid

**Review by nicolaslevy — CHANGES_REQUESTED** [review]
We should add something in the README explaining how to retrieve sessionid and contentid

File: README.md [hunk 1]
```diff
@@ -144,7 +143,8 @@ You can also use the plugin function ````SetId()```` to set a Session ID or a Co
 ```js
 player.cmcd().setId({sid: 'SessionID', cid: 'ContentID'});
 ```
-  
+> **_NOTE:_**  If you don't provide a SID and CID, the plugin will automatically generate them (A hash code for CID and a UUID for SID).
+
```
Comment (nicolaslevy) [inline README.md:147]: and how do I know those values? ;)  player.cmcd().sid ? player.cmcd().getId().sid ?

**Review by nicolaslevy — CHANGES_REQUESTED** [review]

**Review by nicolaslevy — APPROVED** [review]
Great job @coatesjuan :clap:

**Pull request merged**